app/controllers/api/revoke_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  class RevokeController < BaseController
+    def validate_method_action(_method_name, action_name)
+      super(:put, action_name)
+    end
+
+    def target_user
+      userid = params[:userid]
+      if userid
+        user_object = User.find_by(:userid => userid)
+        raise BadRequestError, "Unable to find user #{userid}" unless user_object
+
+        filtered_user = Rbac::Filterer.filtered_object(user_object)
+        raise ForbiddenError, "The user is not authorized for this task or item." unless filtered_user
+
+        filtered_user
+      else
+        User.current_user
```

Comment:
I'm thinking that the current user should be able to revoke his own sessions regardless of RBAC, just like he'd be able to log himself out. Do you think we should skip RBAR for this case?

I'm also thinking this would be the most common way this API will be accessed

Reply:
yes I think also that we don't need to do RBAC check on current user. 

Newly I added here `role_allows?` check for product feature `revoke_current_user_sessions` and
thanks to that we can distinguish privileges
for revoking current user's sessions and 
for revoking sessions of requester users from params. 

thanks